src/script/page/MainContent/panels/Collection/CollectionSection.test.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {render, fireEvent} from '@testing-library/react';
+
+import CollectionSection from './CollectionSection';
+import {ContentMessage} from '../../../../entity/message/ContentMessage';
+import {createRandomUuid} from 'Util/util';
+
+const NUMBER_OF_ASSETS = 5;
+
+const messages = new Array(NUMBER_OF_ASSETS).fill(null).map(() => new ContentMessage(createRandomUuid()));
+
+const getDefaultProps = ({limit}: {limit: number}) => ({
```

Comment:
WDYT? You have only 1 param there :)

```suggestion
const getDefaultProps = (limit: number) => ({
```